src/go/k8s/tests/e2e/produce-consume/01-redpanda-cluster.yaml

```diff
@@ -12,10 +12,10 @@ spec:
   resources:
     requests:
       cpu: 1
-      memory: 2Gi
+      memory: 100Mi
```

Comment:
this is unrelated, right? I have that in my PR as well. Perhaps it should at least go to its own commit?

Reply:
No, you have `100M` I introduce `100Mi`. As per https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory kubernetes provides Mi and M suffixes which seastar doesn't understand. Reference https://github.com/scylladb/seastar/blob/c0d6e042a31c6165d59a581e5802a3d33c4ba33b/src/util/conversions.cc#L32-L47

Reply:
Does this run? I thought redpanda had a check that it will not start unless it has at least 1GB of memory available to it.

Reply:
Whilst we're here, I'll ask: can the `initContainer` get the same resources as the application container? Without resource limits the POD is `Burstable` rather than `Guaranteed`.

Reply:
@rkruze in the Developer mode = true it's possible. Our Custom Resource Definition allows to schedule pandas in developer mode. Reference [Configuration definition](https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go#L79)
@BenPope Agree. In different PR I will address this. Will measure how much resources `rpk` need for simple configuration changes. IMO no more than 100m CPU and 100Mi memory.

Reply:
When implementing the initial `statefulSet`, I came to the conclusion that there's nothing detrimental in just making it the same as the application container, they run one after the other, so scheduling shouldn't be a problem.

Reply:
@alenkacz This is deliberate change in order to verify that `100Mi` is correctly translated into [arg](https://github.com/vectorizedio/redpanda/pull/602/files#diff-5785f46b22d91978ea450fdfb28b84a7859c50556e356baea7a8680443090003R16). 
I don't see the reason to split feature from its test.